fleet-examples/fleets/opni/opni/fleet.yaml

```diff
@@ -0,0 +1,17 @@
+defaultNamespace: opni
+helm:
+  releaseName: opni
+  chart: "https://github.com/rancher/opni/raw/charts-repo/assets/opni/opni-0.10.0.tgz"
+  repo: ""
+targetCustomizations:
+- name: local
+  clusterSelector:
+    matchLabels:
+      management.cattle.io/cluster-display-name: local
+  helm:
+    values:
+      gateway:
+        hostname: opni-local
+        serviceType: NodePort
```

Comment:
I am not that familiar with fleet yet, but will it make more sense to template the values for opni? That way a user could specify configurations that are specific for the setup of Opni that he/she would like to run.

e.g. Having an Opni deployment that uses `openid` for monitoring authentication. WDYT?

Reply:
Yes and no :)

Fleet allows you to customize the deployment per cluster or group of clusters by using `targetCustomizations`, so you can _template_ the values depending on the cluster https://fleet.rancher.io/ref-fleet-yaml

The idea for this PR is to provide a simple example to deploy Opni and if a more complex deployment is needed, then use the above customizations.

Reply:
Okay, got it. Thanks for the clarification. Looks good to me then :)